docs/high-availability.md

```diff
@@ -0,0 +1,126 @@
+# High Availability Kubernetes Clusters
+
+## Introduction
+This document describes how to build a high-available Kubernetes cluster.  This is a fairly advanced topic, generally useful to
+people who need to set up a higly availble, redundant Kubernetes cluster for production workloads.  Users who merely want to
+experiment with Kubernetes are encouraged to try [Google Container Engine](https://cloud.google.com/container-engine/) or the
+simple [Docker based single node cluster instructions](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/docker.md)
+
+## Overview
+Setting up a truly reliable, highly available distributed system requires a number of distinct steps, it is akin to
+wearing underwear, pants, a belt, suspenders, another pair of underwear, and another pair of pants.  We go into each
+of these steps in detail, but a summary is given here to help guide and orient the user.
+
+The steps involved are as follows:
+   * Setting up a redundant, reliable storage layer
+   * Installing a health-checking daemon runner
+   * Turning up a clustered instance of the etcd key/value store
+   * Starting a replicated, load balanced kubernetes API servers
+   * Setting up master-elected kubernetes components
+
+Ready? Let's get started.
+
+## Initial set-up.
+The remainder of this guide assumes that you are setting up a 3-node clustered master, where each machine is running some flavor of Linux.
+Examples in the guide are given for Debian distributions, but they should be easily adaptable to other distributions.
+
+## Reliable storage
```

Comment:
Addressed, modified using reliable storage as "extra redundancy" rather than the default.